gax-java/gax/src/main/java/com/google/api/gax/rpc/ServerStream.java

```diff
@@ -89,6 +91,10 @@ public Iterator<V> iterator() {
     return iterator;
   }
 
+  public Stream<V> stream() {
+    return StreamSupport.stream(this.spliterator(), false);
```

Comment:
I see that the second argument is set to false, which means this is a sequential stream, do we want to expose another `parallelStream` method as well?

Reply:
I'm not sure whether a deterministic result is required as parallel stream can randomize the result.